sdk/src/main/java/io/radar/sdk/RadarLocationManager.kt

```diff
@@ -538,6 +549,11 @@ internal class RadarLocationManager(
                     if (foregroundService != null && foregroundService.updatesOnly) {
                         locationManager.stopForegroundService()
                     }
+
+                    if (trackingOptions != null && RadarSettings.getServerTrackingOptions(context)) {
```

Comment:
after a POST /track request completes, we update tracking options.

it's worth noting, we don't do this for GET /config, except when we initially call the new parameterless `startTracking` method

Reply:
comment here too?